v3/util/etcdctl-auth-root.sh

```diff
@@ -0,0 +1,71 @@
+
+#!/bin/bash
+
+source /etc/environment
+
+#Disable authentication
+
+#curl  -L http://127.0.0.1:2379/v2/auth/enable -XDELETE
+
+etcdctlrootusername=$(etcdctl get /etcdctl/config/root-user)
+etcdctlrootpassword=$(etcdctl get /etcdctl/config/root-password)
+etcdctletcdreadusername=$(etcdctl get /etcdctl/config/read-user)
+etcdctletcdreadpassword=$(etcdctl get /etcdctl/config/read-password)
+etcdctletcdreadwriteusername=$(etcdctl get /etcdctl/config/write-user)
+etcdctletcdreadwritepassword=$(etcdctl get /etcdctl/config/write-password)
+
+#add a root user
+
+echo '{"user":"'$(eval echo $etcdctlrootusername)'", "password":"'$(eval echo $etcdctlrootpassword)'"}' > /home/core/mesos-systemd/v3/util/root.json
```

Comment:
Change `/home/core/mesos-systemd/v3/util/root.json` to `/opt/etcdctl/root-user.json`